modules/bookList.js

```diff
@@ -0,0 +1,48 @@
+const booksContainer = document.querySelector('.books');
+
+export default class BookList {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(book) {
+    if (localStorage.getItem('localData') === null) {
+      localStorage.setItem('localData', JSON.stringify([]));
+      this.books.push(book);
+      localStorage.setItem('localData', JSON.stringify(this.books));
+      window.location = window.location.pathname;
+    } else {
+      this.books = JSON.parse(localStorage.getItem('localData'));
+      this.books.push(book);
+      localStorage.setItem('localData', JSON.stringify(this.books));
+      window.location = window.location.pathname;
+    }
+  }
+
+  removeBook(title) {
+    this.books = JSON.parse(localStorage.getItem('localData'));
+    this.books.forEach((book, index) => {
+      if (book.title === title) {
+        this.books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('localData', JSON.stringify(this.books));
+    window.location = window.location.pathname;
+  }
```

Comment:
- [ ] When removing a duplicate book it removes both instances instead of just 1 please make sure only to remove the book the user actually wants to delete. you can do this by deleting based on ID.